virtual-setup/zoom-procedures.md

```diff
@@ -1 +1,197 @@
-## Zoom procedures for CCDL virtual workshops
+# Zoom Procedures for CCDL Virtual Workshops
+
+We use [Zoom](https://zoom.us/) meetings for conducting CCDL virtual workshops.
+In this document, we will cover the procedures for joining a Zoom call and using various features for instruction and consultation days.
+
+#### Instructions for setting up Zoom
+
+We have instructions for setting up Zoom for your operating system available.
+
+* [Linux](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/linux-instructions.md#zoom)
+* [Mac](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/mac-instructions.md#zoom-installation-and-setup)
+* [Windows](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/windows-instructions.md#zoom)
+
+#### Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [General tips for using Zoom during workshops](#general-tips-for-using-zoom-during-workshops)
+  - [Audio](#audio)
+  - [Video](#video)
+  - [Interacting with instructors and providing feedback](#interacting-with-instructors-and-providing-feedback)
+    - [Participant pane](#participant-pane)
+    - [Chat](#chat)
+- [Joining a Zoom call](#joining-a-zoom-call)
+  - [To join a call](#to-join-a-call)
+- [Zoom recordings](#zoom-recordings)
+- [Using Zoom breakout rooms](#using-zoom-breakout-rooms)
+  - [Asking for 1:1 help during instruction](#asking-for-11-help-during-instruction)
+  - [Joining a breakout room](#joining-a-breakout-room)
+  - [Sharing your screen in Zoom breakout rooms](#sharing-your-screen-in-zoom-breakout-rooms)
+  - [Returning to the main session](#returning-to-the-main-session)
+  - [Consultation days](#consultation-days)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## General tips for using Zoom during workshops
+
+### Audio
+
+Please make sure your microphone is muted when joining the call and when you are to minimize background noise during instruction.
+You can toggle mute on and off using the `Mute` (or `Unmute` if you are muted) icon on the left side of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-mute.png>
+
+Where possible, please utilize headphones throughout training to minimize audio feedback.
+
+### Video
+
+If you are experiencing issues with the call quality, turning off your video may help.
```

Comment:
This is correct, right?
```suggestion
If you are experiencing issues with the call quality, turning off your video may help.
This will not affect your ability to see the instructors or content, only outgoing video.
```

Reply:
Yep. Will make the change.